### DISCUSSION TITLE
AsyncCatcher can't be disabled, can you make it able to be disabled?

### DISCUSSION THREAD

**Discussion by Glomzzz** [PurpurMC/Purpur#1216] [__Dismissed__] [answered]
As you see, many plugins use some "safe" async opreation, such as getEntity, but in Purpur, no async operation is allowed.
Which means, many coder must switch their way of operation from async to sync, which means more performance loss in the main thread.
Tips: let the AsyncCatcher.enabled worked!

**Comment by BillyGalbreath** [comment, marked as answer]
No.
